apps/self-hosted/hosting/scripts/hivesigner-redirect-uris.py

```
@@ -131,6 +271,113 @@ def write_private(contents: str, path: str | None) -> str:
+    Update the account's posting_json_metadata.
+
+    account_update2 leaves a field alone when it is sent empty, so json_metadata
+    is passed as "" and only the posting copy is written. lighthive serializes
+    through the node's own get_transaction_hex, so this needs no client-side
+    operation table that could fall behind the chain.
+    """
+    try:
+        from lighthive.client import Client
+        from lighthive.datastructures import Operation
+    except ImportError:  # pragma: no cover - depends on the operator's box
+        raise SystemExit("lighthive is required to broadcast: pip install lighthive")
```

Ruff (0.16.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)